frontend/yarn.lock

```diff
@@ -0,0 +1,5378 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Does this one even belong in VC? A [google search](https://classic.yarnpkg.com/en/docs/yarn-lock/) suggests that it does, but it seems super bloated and I'd be happy to remove it.

Reply:
Yup, this is the file that locks package versions down so you don't have unintentional upgrades - and yeah, this is usually checked in